tests/ui/proc-macro/attribute.rs

```diff
@@ -10,97 +10,132 @@ use proc_macro::*;
 //~^ ERROR malformed `proc_macro_derive` attribute
 //~| NOTE expected this to be a list
 //~| NOTE for more information, visit
-pub fn foo1(input: TokenStream) -> TokenStream { input }
+pub fn foo1(input: TokenStream) -> TokenStream {
+    input
+}
```

Comment:
Can you undo the reformatting of this file? Files within `tests/ui/` aren't auto-formatted, and it's just churn that obscures the actual change in this commit.